code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1457,9 +1457,13 @@
 
 /mob/living/carbon/human/proc/get_eye_shine() //Referenced cult constructs for shining in the dark. Needs to be above lighting effects such as shading.
 	var/obj/item/organ/external/head/head_organ = get_organ("head")
-	var/datum/sprite_accessory/hair/hair_style = GLOB.hair_styles_full_list[head_organ.h_style]
-	var/icon/hair = new /icon("icon" = hair_style.icon, "icon_state" = "[hair_style.icon_state]_s")
-	var/mutable_appearance/MA = mutable_appearance(get_icon_difference(get_eyecon(), hair), layer = ABOVE_LIGHTING_LAYER)
+	var/datum/sprite_accessory/hair_style = GLOB.hair_styles_full_list[head_organ.h_style]
```

Comment:
👁️ 👄 👁️ 

That looks like the wrong datum level. Inb4 reviewer says: did you mean: `var/datum/sprite_accessory/hair/hair_style`? ;--; yes, yes I did. I'm sorry.